Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -185,6 +199,7 @@ private void OnInteractUsing(Entity<ReagentGrinderComponent> entity, ref Interac
 
         private void UpdateUiState(EntityUid uid)
         {
+            var comp = Comp<ReagentGrinderComponent>(uid);
```

Comment:
This component isn't guaranteed to exist and should really be a resolve instead.

Reply:
Fixed